### ISSUE TITLE
Upload to youtube feature

### ISSUE THREAD

**Issue by blaues0cke** [blaues0cke/PiSleepTalk#17]
Would be great to be able to upload videos automatically to youtube.

**Comment by ottojo** [comment]
https://developers.google.com/youtube/v3/docs/videos/insert

**Comment by blaues0cke** [comment]
Yeah, thinks its also a good idea to use some projects that already do that:
- https://code.google.com/p/youtube-upload/wiki/Readme
- http://stackoverflow.com/questions/8921496/shell-script-to-download-youtube-files-from-playlist
